packages/vue-query/tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "rootDir": "./src",
+    "outDir": "./build/lib",
+    "tsBuildInfoFile": "./build/.tsbuildinfo"
+  },
+  "include": ["src"],
+  "exclude": ["src/__tests__"],
```

we have removed this lately because it causes tons of errors in local IDEs. Can we remove it here as well?

```suggestion
  },
  "include": ["src"],
```

---

Yeah, since i was using `tsc` to build vue-query before it was necessary. But now it's not 👍